### ISSUE TITLE
`no_answer` status from callbacks of `call` type

### ISSUE THREAD

**Issue by katchengli** [messagebird/java-rest-api#94]
Hello,

We're receiving callbacks of type call with this format:
```
{"timestamp":"2020-02-13T20:14:00.000Z",
"items":[
{"type":"call",
"event":"callUpdated",
"payload":{
"id":"<redacted>",
"status":"no_answer",
"source":"<redacted>",
"destination":"<redacted>",
"webhook":{"url":"<redacted>"},
"createdAt":"2020-02-13T20:00:00Z",
"updatedAt":"2020-02-13T20:10:00Z",
"endedAt":"2020-02-13T20:11:00Z"
}}]}
```
However, the `VoiceCallStatus` using by the `VoiceCallData` does not support the `no_answer` status. The possibility of that status also seems undocumented: https://developers.messagebird.com/api/voice-calling#calls

Relevant code:
https://github.com/messagebird/java-rest-api/blob/master/api/src/main/java/com/messagebird/objects/voicecalls/VoiceCallData.java#L35
https://github.com/messagebird/java-rest-api/blob/master/api/src/main/java/com/messagebird/objects/voicecalls/VoiceCallStatus.java#L5

Can we add it to the API?

**Comment by denizkilic** [comment]
Hi @katchengli,
Thanks for reporting this issue. We are going to update the VoiceCallStatus object. I will inform you after we released a new version.

**Comment by denizkilic** [comment]
Hi @katchengli,
This issue was fixed by the latest version (Release 3.0.9) We added three more enums which are "failed", "busy" and "no_answer" on VoiceCallStatus. Thank you and we are welcoming more issues.

**Comment by katchengli** [comment]
@denizkilic Thank you for the quick fix!
